### PR TITLE
[#37] fixes previous submission not loading from cache

### LINKED ISSUE #37: Previous submission not loading from cache
#### Software and hardware versions 
Collect v2.6.0

#### Problem description
Previous submission are not being stored offline in the app

#### Steps to reproduce the problem
- open a form 
- make a submission
- tap on view submission button on that form (the submission you just made should appear)
- Turn off internet (WIFI/data)
- Tap on view submission button on that form (the submission you just made should appear)

#### Expected behavior
If previous submission has been opened for a form, it should be available even when the device is offline

### PR DESCRIPTION
**Pull request by konishon**
Closes #37

### CODE REVIEW

File: collect_app/src/main/java/org/bcss/collect/naxa/submissions/PreviousSubmissionListActivity.java [hunk 1]
```diff
@@ -165,7 +167,7 @@ public void onResponse(Call<FormHistoryResponse> call, Response<FormHistoryRespo
 
                 FormHistoryResponse formHistoryResponse = response.body();
 
-                if (formHistoryResponse == null || formHistoryResponse.getResults().size() <= 0) {
+                if (formHistoryResponse.getResults().size() <= 0) {
```
Comment (hereyubaraj) [inline collect_app/src/main/java/org/bcss/collect/naxa/submissions/PreviousSubmissionListActivity.java:170]: `formHistoryResponse.getResults().size()` throws error if `response.body()` is `null`

File: collect_app/src/main/java/org/bcss/collect/naxa/submissions/PreviousSubmissionListActivity.java [hunk 2]
```diff
@@ -133,6 +130,11 @@ public void onClick(View view) {
                         lazyLoadFirstPage(TimeUnit.SECONDS.toMillis(1));
                     }
                 });
+
+        setupPagination(listFormHistory);
+        cardSubmissionInfo.setVisibility(View.GONE);
+        progressBar.setVisibility(View.VISIBLE);
```
Comment (hereyubaraj) [inline collect_app/src/main/java/org/bcss/collect/naxa/submissions/PreviousSubmissionListActivity.java:136]: These lines of code can be removed as it is always called in `onCreate()`

**Review by hereyubaraj — COMMENTED** [review]
@nishontan please check the review

**Comment by konishon** [comment]
@hereyubaraj 
are comments working for you?

**Pull request merged**